src/CommunityToolkit.Maui/Views/Popup/Popup.shared.cs

```diff
@@ -269,7 +290,7 @@ static void OnColorChanged(BindableObject bindable, object oldValue, object newV
 
 	void IPopup.OnOpened() => OnOpened();
 
-	void IPopup.OnDismissedByTappingOutsideOfPopup() => OnDismissedByTappingOutsideOfPopup();
+	async void IPopup.OnDismissedByTappingOutsideOfPopup() => await OnDismissedByTappingOutsideOfPopup();
```

Comment:
Should it be task?

Reply:
I thought about this! 

But ultimately, I decided that `IPopup.OnClosed()`, `IPopup.OnOpened()` and `IPopup.OnDismissedByTappingOutsideOfPopup()` are basically event handlers and can be fire-and-forget.

The biggest downside to changing `IPopup.OnDismissedByTappingOutsideOfPopup()` to return a `Task` is that it'd be a breaking change in the API. Plus, if other libraries want to create their own `IPopup`, they'd then be forced to return a `Task` whether they use it or not.